test/parser.coffee

```diff
@@ -52,7 +52,7 @@ test "new calls have a range including the new", ->
     eq node.locationData.last_column, lastColumn
 
   [assign] = block.expressions
-  outerCall = assign.value
+  outerCall = assign.value.base
```

Comment:
Can you please explain the changes in this file?

Reply:
The `new` `Call` now ends up wrapped in a `Value` (related to the `new`-wrangling in `Op::constructor()`). I guess in theory the test could use `.unwrap()` to avoid explicitly "hardcoding" whether there's a `Value` wrapper or not, but there were only two tests that needed updating